### ISSUE TITLE
Implement editing profile picture frontend and backend

### ISSUE THREAD

**Issue by johntolo** [cse442-at-ub/project_s23-cinco#118]
Test 1:
go to website url: https://www-student.cse.buffalo.edu/CSE442-542/2023-Spring/cse-442b/build
login with any credentials
click on to Profile tab
ensure that it shows a current profile picture
click on Edit Profile Pic and click any picture
Ensure the profile picture now changes to the one you picked (see screenshot)

![image.png](https://images.zenhubusercontent.com/63e2cf463bc662d519ec1cc1/473972ab-263f-41fa-84c7-b23b03135147)

**Comment by junqiwu02** [comment]
rephrase it to ensure the pfp displays your current one
